pkg/queue/protobuf_stats_reporter_test.go

```diff
@@ -24,18 +24,28 @@ import (
 	"testing"
 	"time"
 
+	"github.com/google/go-cmp/cmp/cmpopts"
+
+	"github.com/google/go-cmp/cmp"
+
 	"knative.dev/serving/pkg/autoscaler/metrics"
 	"knative.dev/serving/pkg/network"
 )
 
-var testStat = metrics.Stat{
-	PodName:                          "testPod",
-	AverageConcurrentRequests:        5.0,
-	AverageProxiedConcurrentRequests: 5.0,
-	ProxiedRequestCount:              100.0,
-	RequestCount:                     100.0,
-	ProcessUptime:                    20.0,
-}
+var (
+	testStat = metrics.Stat{
+		PodName:                          "testPod",
+		AverageConcurrentRequests:        5.0,
+		AverageProxiedConcurrentRequests: 5.0,
+		ProxiedRequestCount:              100.0,
+		RequestCount:                     100.0,
+		ProcessUptime:                    20.0,
+	}
+
+	ignoreStuff = cmp.Options{
+		cmpopts.IgnoreFields(testStat, "ProcessUptime"),
+	}
```

Comment:
Why do we ignore this? The old test code had a hardcoded `==` check here :thinking:

Reply:
Misread about the time discussion above.